src/lib.rs

```diff
@@ -1049,7 +1050,7 @@ impl GooseAttack {
                     info!("waiting for clients to exit");
                 }
                 for client in clients {
-                    let _ = client.join();
+                    let _ = client.await;
```

Comment:
Should you not join all clients together instead of executing them in serial?

Reply:
we're now using `futures::future::join_all()`